climesync.py

```diff
@@ -702,31 +767,32 @@ def menu():
 
 
 def main():
-    global arg_username, arg_password, timesync_url, ts
-
     parser = argparse.ArgumentParser()
     parser.add_argument("-c", "--connect", help="connect to a timesync server")
     parser.add_argument("-u", "--username", help="specify your username")
     parser.add_argument("-p", "--password", help="specify your password")
 
     # Command line arguments
     args = parser.parse_args()
+    url = args.connect
+    user = args.username
+    password = args.password
 
-    if args.connect:
-        timesync_url = args.connect
+    # Config file
+    config_dict = {}
```

Comment:
You don't need this line since you set `config_dict` in both the `try` and `except` blocks below.